src/rulesets/oas/index.ts

```diff
@@ -1,5 +1,5 @@
 import { FunctionCollection } from '../../types';
-import { readRulesFromRulesets } from '../reader';
+import { readRuleset } from '../reader';
 
 export const commonOasFunctions = (): FunctionCollection => {
```

Comment:
This definitely seems like a thing we should be able to delete?

Reply:
I am more than happy to remove it, but it will be breaking.
If there is a code relying on it (there certainly is - it must be used if you use Spectral programmatically), imports will break, hence the code will need to be updated.